Tel 118 get recurring tasks

mrcpvn opened the pull request.

WW-CICD commented:
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsAdminRecurringTaskMonitor-CodeBuildProject
* Commit ID: 75c7d84e5349880d9bb0afe6616bef27400933df
* Result: FAILED
* [Build Logs](https://lrp0k39ym9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c30ba8aa-e315-42f9-ae93-2cab69b7daae%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

WW-CICD commented:
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsAdminRecurringTaskMonitor-CodeBuildProject
* Commit ID: aed4f66f9f69ae74d215199e5b3ae6b7c401a2bc
* Result: FAILED
* [Build Logs](https://lrp0k39ym9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=42f17511-c0c6-433c-9369-fce8ef8263a8%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

WW-CICD commented:
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsAdminRecurringTaskMonitor-CodeBuildProject
* Commit ID: a4afcbceb61746fb7c396f8264dbc281d6bd442a
* Result: SUCCEEDED
* [Build Logs](https://lrp0k39ym9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6fa53606-0e87-4d49-be18-ca9e27b4acee%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

mrcpvn commented:
Sorry this has been committed by mistake. I agree that we shouldn't commit
this stuff.
I removed the .lambda-test-tool folder from the repo and added the
.lambda-test-tool folder to .gitignore.

On Wed, Mar 4, 2020 at 3:22 PM David Witzling <notifications@github.com>
wrote:

> *@davidw-foxtrax* approved this pull request.
>
> I notice that the .lambda-test-tool/SavedRequests/alb.json has been
> committed. Is this intentional?
>
> I would wonder if we should add .lambda-test-tool to the .gitignore file
> instead.
>
> My thinking is that this is a manual testing tool and we shouldn't be
> saving these tests into the repo. The queries impact local data, and saving
> them implies that a shared system exist which does not.
>
> On the other hand, they could be convenient to have around, as long as we
> are careful not to commit requests that are too specific. If you think we
> should be storing them, I'm ok with it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorkWave/WWTelematics.Admin.RecurringTaskMonitor/pull/3?email_source=notifications&email_token=ACWJQCSSJHMNP3UIZ37LZB3RFZPZHA5CNFSM4LBAPIR2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCX53EZA#pullrequestreview-368816740>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACWJQCU56CQOQSMYMWQ7OE3RFZPZHANCNFSM4LBAPIRQ>
> .
>


-- 
*Marco Piovan*

Senior System Engineer
mpiovan@workwave.com | workwave.com

Via Luigi Da Porto, 2/C - 37122 Verona (VR) - Italy

​
<https://www.workwave.com/>    <https://www.facebook.com/WorkWaveCo>
<https://www.linkedin.com/company/workwave>   <https://twitter.com/WorkWave>

WW-CICD commented:
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsAdminRecurringTaskMonitor-CodeBuildProject
* Commit ID: b8876ad5653328ffd02efce37d6f63832528e943
* Result: SUCCEEDED
* [Build Logs](https://lrp0k39ym9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e3f14aa5-3f23-4b1c-980c-3fd72e387dc0%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

WW-CICD commented:
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsAdminRecurringTaskMonitor-CodeBuildProject
* Commit ID: d44cfab7def332a607d52f639b5fed34ab5b0176
* Result: SUCCEEDED
* [Build Logs](https://lrp0k39ym9.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=98872bb7-63de-4742-bc93-f16d4062d6f4%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->
